lib/components/mobile/navigation-bar.js

```diff
@@ -111,6 +114,9 @@ class MobileNavigationBar extends Component {
             )}
           </MobileBar>
         </Navbar>
+        <NetworkConnectionBanner
```

Comment:
I think this is causing the percy tests to fail on mobile nearby view? not sure what else would cause the issues percy has.